analytic_engine/src/sst/builder.rs

```diff
@@ -35,6 +35,16 @@ pub mod error {
             backtrace: Backtrace,
         },
 
+        #[snafu(display(
+            "Hybrid format don't support variable length type, type:{}.\nBacktrace:\n{}",
+            type_str,
+            backtrace
+        ))]
+        VariableLengthType {
+            type_str: String,
```

Comment:
How about `type_name`? The `str` suffix is verbose because we all know its type.